siatest/testfile.go

```diff
@@ -0,0 +1,60 @@
+package siatest
+
+import (
+	"bytes"
+	"io/ioutil"
+	"math"
+	"path/filepath"
+	"strconv"
+
+	"github.com/NebulousLabs/Sia/build"
+	"github.com/NebulousLabs/fastrand"
+)
+
+type (
+	// TestFile is a helper struct to easily create files on disk that are
+	// ready to use with the TestNode.
+	TestFile struct {
+		path     string
+		fileName string
+		siaPath  string
+	}
+)
+
+// Bytes returns the contents of the TestFile
+func (tf *TestFile) Bytes() []byte {
+	data, err := ioutil.ReadFile(tf.path)
+	if err != nil {
+		println(err)
+		return []byte{}
+	}
+	return data
+}
+
+// Compare is a convenience function that compares the contents of two
+// TestFiles on disk. Its behavior is similar to bytes.Compare.
+func (tf *TestFile) Compare(tf2 *TestFile) int {
+	tfData, err := ioutil.ReadFile(tf.path)
+	tf2Data, err2 := ioutil.ReadFile(tf2.path)
+	if err != nil || err2 != nil {
+		// Print error and return a mismatch instead of returning the error.
+		// This should be sufficient for a testing environment and makes for
+		// cleaner tests.
+		println(build.ComposeErrors(err, err2))
```

Comment:
You want to return an error so that the test can fail. We definitely don't read through the print statements of travis and appveyor every time that the build passes.